osf/external/gravy_valet/request_helpers.py

```diff
@@ -154,13 +192,159 @@ def _make_gv_request(
+        gv_response = _get_gv_response(
+            auth=auth,
+            addon=addon,
+            project=project,
+            list_id=list_id
+        )
+        if gv_response.status_code == 201:
+            attributes_dict = gv_response.json()['data']['attributes']
+            items = attributes_dict.get('operation_result').get('items')
+            for item in items:
+                item_id = item.get('item_id', '')
+                kind = CITATION_ITEM_TYPE_ALIASES.get(item.get('item_type', ''))
+                if 'csl' in item.keys():
+                    change_response = item
+                    change_response['kind'] = kind
+                    change_response['id'] = item_id
+                else:
+                    change_response = {
+                        'data': {
+                            'addon': addon_short_name,
+                            'kind': kind,
+                            'id': item_id,
+                            'name': item.get('item_name', ''),
+                            'path': item.get('item_path', '/'),
+                            'parent_list_id': item.get('parent_list_id', 'ROOT'),
+                            'provider_list_id': item_id,
+                        },
+                        'kind': kind,
+                        'name': item.get('item_name', ''),
+                        'id': item_id,
+                        'urls': {
+                            'fetch': f'/api/v1/project/{project._id}/{addon_short_name}/citations/{item_id}/',
+                        }
+                    }
+                response['contents'].append(change_response)
+    return response
+
+
+def get_zotero_library_list(auth, request, addon_short_name):
+    from osf.models import Node
+
+    changed_response = []
+    project = Node.objects.filter(guids___id__in=[request.view_args.get('pid')]).first()
+    citation_url_list = get_citation_url_list(
+        auth=auth,
+        request=request,
+        addon_short_name=addon_short_name,
+        project=project)
+    for addon in citation_url_list:
+        gv_response = _get_gv_response(
+            auth=auth,
+            addon=addon,
+            project=project,
+            list_id='ROOT'
+        )
+        if gv_response.status_code == 201:
+            attributes_dict = gv_response.json()['data']['attributes']
+            items = attributes_dict.get('operation_result').get('items')
+            for item in items:
+                item_id = item.get('item_id', '')
+                changed_response.append({
+                    'addon': addon_short_name,
+                    'kind': item.get('item_type', '').lower(),
+                    'id': item_id,
+                    'name': item.get('item_name', ''),
+                    'path': item.get('item_path', ''),
+                    'parent_list_id': None,
+                    'provider_list_id': item_id,
+                })
+    return changed_response
```

Comment:
This request is no longer needed when `gravy_waffle` is enabled, but I'd advice you to test this out before making changes

Reply:
I tried to remove library_list but too many errors appeared

Reply:
Have you tried removing GV library list only? 
And may you showcase which errors have appeared, please
?